src/main/java/org/folio/inventory/resources/Items.java

```diff
@@ -93,10 +95,15 @@ public void register(Router router) {
     router.put(RELATIVE_ITEMS_PATH_ID).handler(this::update);
     router.delete(RELATIVE_ITEMS_PATH_ID).handler(this::deleteById);
 
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-in-process")
-      .handler(handle(this::markAsInProcess));
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-in-process-non-requestable")
-      .handler(handle(this::markAsInProcessNonRequestable));
+    Arrays.stream(ItemStatusName.values())
+      .map(x->ItemStatusURL.getUrlForItemStatusName(x))
```

Comment:
I find it easier to understand expressions like this when the parameters have meaningful names. Alternatively, use a method reference instead.

Similarly for the optional value presence filter below this.

```suggestion
      .map(statusName -> ItemStatusURL.getUrlForItemStatusName(statusName))
```

```suggestion
      .map(ItemStatusURL::getUrlForItemStatusName)
```